blocks/page_list/templates/itinerary/ItineraryStore.js

```diff
@@ -1,59 +1,122 @@
 import { dispatch, register } from './ItineraryDispatcher';
 import { EventEmitter } from 'events';
 import ItineraryConstants from './ItineraryConstants';
```

Comment:
Once my above comment for the ItineraryConstants goes in, you'd be able to use it here like:
```
import {Actions} from './ItineraryConstants';
// ...
case Actions.UPDATE_TITLE:
```